asdf/external_reference.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import, division, unicode_literals, print_function
```

Comment:
Minor comment but we'll be removing Py2 support in the next major release, so no need to include this here.